### DISCUSSION TITLE
Existing Private GitHub Repository

### DISCUSSION THREAD

**Discussion by vahideabdi** [community/community#73404] [Repositories] [answered]
### Select Topic Area

Question

### Body

When I created a private repository, I didn't have the education plan. So my Repository did not have a wiki tab. Now I upgraded my plan to an education plan and I can't or don't know how to add the wiki tab for that repository.. or do I have to create a new repository and transfer all data in order to get the wiki tab?

**Comment by mehulrajdev** [comment, marked as answer]
Check the settings of your repository. If you scroll down to the "Features" section there should be checkboxes for wiki and a bunch of other things.

**Comment by esurov** [comment]
Dear @vahideabdi ,
You don't need to create a new repository and transfer your data to enable the Wiki tab for your private GitHub repository after upgrading to an education plan. Instead, you can simply enable the Wiki feature for your existing repository. Here's how you can do it:

1. Go to your GitHub repository.

2. Click on the "Settings" tab at the right end of the repository's menu bar.

3. In the left sidebar, click on "Options."

4. Scroll down to the "Features" section.

5. You should see a "Wikis" option with a checkbox. Simply check the box to enable the Wiki feature for your repository.

6. Scroll to the bottom of the page and click the "Save changes" button to apply the changes.

After following these steps, the Wiki tab should appear in your repository, and you can start using it to create and edit Wiki pages.

Keep in mind that enabling the Wiki feature is a repository-level setting, so it applies to the specific repository you are configuring. You can have some repositories with Wikis enabled and others without, depending on your needs.
